src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -28,12 +28,12 @@
         -error   -->
     <!-- The default level of a check is "error". -->
     <!-- See http://checkstyle.sourceforge.net/config.html#Severity -->
-    <property name="severity" value="info" />
-    <property name="ignoreLines" value="1,2,3"/>
+    <property name="severity" value="error" />
+    <property name="ignoreLines" value="2"/>
```

Comment:
The line containing the year, unfortunately it is different in the openhab-core and openhab2-addons repo, so no idea how to overcome this for now ...